docs/scanner-pools.md

```diff
@@ -0,0 +1,182 @@
+
+It is up to the pool owner to allocate more from owner stake or more from the delegators so the share of the rewards can be adjusted accordingly.
+
+For each node in a pool:
+
+- The operator needs to allocate at least 2500 FORT.
+- The total capacity is 15000 FORT.
+
+When you deposit more stake on your pool as the operator and then register a node, your stake is allocated automatically. It is up to you to decide how much of owner and delegated stake to be allocated.
+
+!!! warning "Risk of shutting down the pool"
+    After unallocating owner stake as the operator, if allocated owner stake does not satisfy `node_count × min_stake_per_scan_node`, the pool faces the risk of shutting down and will generate no rewards! Please take into account the warnings and your calculations before proceeding.
+
+    E.g. for a pool with two nodes, 4k FORT stake on the pool means that both of the nodes in the pool are non-operational until it can be topped up to 5k (`2500 × 2`) and allocation of owner stake is adjusted accordingly.
+
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on the node pool you would like to adjust the stake allocation
+
+![My Node Pools List](MyNodePoolsList.png)
+
+- Click on "Allocate Stake"
+
+![Allocate Stake On Node](AllocateStakeOnNode.png)
+
+- From here you can either allocate or unallocate stake. To adjust owner stake make sure the subject is *Owner*
+
+![Allocate Modal](AllocationModal.png)
+
+- To adjust the stake of delegators select the *Delegators* subject
+
+![AllocateDelegates](AllocateDelegates.png)
+
+## Changing the commission
+
+In addition to the allocation adjustment, operators can decide how much of the delegators' rewards they would like to keep as commission.
+
+!!! important "Phase II feature"
+    These commissions will be effective when Forta App supports claiming the rewards. Meanwhile, the pool owners can still expect delegators to stake on their pools and adjust allocation and commission to get ready for the second phase.
+
+- Click on the node pool you would like to adjust the stake allocation (you must be the owner)
+
+- Click on "Change commission rate"
+
+![Change commission Rate](ChangeCommision.png)
+
+- Enter the commission that you would like to set on your pool
+
+![Update commision](UpdateCommissionModal.png)
+
+## Claiming rewards
+
+!!! important "Phase II feature"
+    This feature that will be available in Forta App in the second phase of the transition to delegated staking. Meanwhile, you can claim pool owner rewards from Polygonscan.
+
+After the rewards are calculated at the end of each epoch, they are transferred to the rewards distributor contract. This contract allows the pool owner and the delegators to claim their rewards.
+
+To claim pool owner rewards over Polygonscan **after Monday 00:00:00 UTC**:
+
+- find out the `RewardsDistributor` contract address from the [smart contract addresses page](smart-contracts.md),
```

Comment:
instead of asking them to go find the address on another page, we should just provide the Polygonscan link directly to the smart contract methods. for example, this link will take me to the `transfer` function of the FORT token contract https://etherscan.io/address/0x41545f8b9472D758bB669ed8EaEEEcD7a9C4Ec29#writeProxyContract#F16

this will reduce the number of steps we need to describe here

Reply:
Yes, I agree. I will do this when we know the contract address.

https://github.com/forta-network/docs/pull/173#discussion_r1108379119

Reply:
Done.